### PR TITLE
[INLONG-7560][Dashboard] When submitting group approval, determine whether to create source and sink

### PR DESCRIPTION
**Pull request by bluewang**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes https://github.com/apache/inlong/issues/7560

### Verifying this change
image

### CODE REVIEW

**Review by healchow — APPROVED** [review]

File: inlong-dashboard/src/locales/en.json [hunk 1]
```diff
@@ -538,6 +538,7 @@
   "pages.GroupDetail.Info.DataStreams": "Data streams",
   "pages.GroupDetail.Info.Submit": "Submit",
   "pages.GroupDetail.Info.SubmittedSuccessfully": "Submitted successfully",
+  "pages.GroupDetail.Info.SubmittedWarn": "Please create data source and sink first",
```
Comment (healchow) [inline inlong-dashboard/src/locales/en.json:541]: What if the user wants to customize push and consume data without source and sink?

Comment (dockerzhang) [reply inlong-dashboard/src/locales/en.json:541]: InLong is not an MQ service, a data stream must contain a source and sink, `auto push` and `auto push` are kinds of source and sink.

Comment (healchow) [reply inlong-dashboard/src/locales/en.json:541]: Okay~

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Comment by leezng** [comment]
Waiting for discussion.

**Review by leezng — APPROVED** [review]

**Pull request merged**